.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```

Comment:
I sure can. I use Windows so line endings are different. I'll add a `.editorconfig` file to enforce line endings.